frontends/web/src/routes/settings/components/device-settings/bip85-setting.tsx

```diff
@@ -0,0 +1,113 @@
+  const { t } = useTranslation();
+  const [status, setStatus] = useState<'idle' | 'info' | 'progress'>('idle');
+
+  if (canBip85) {
+    return (
+      <SettingsItem
+        settingName={t('deviceSettings.expert.showDeterministicEntropy.title')}
+        secondaryText={t('deviceSettings.expert.showDeterministicEntropy.description')}
+        extraComponent={<Warning width={20} height={20} />}
+        displayedValue="Requires newer firmware"
+      />
+    );
+  }
+
+  return (
+    <>
+      <SettingsItem
+        onClick={() => setStatus('info')}
+        settingName={t('deviceSettings.expert.showDeterministicEntropy.title')}
+        secondaryText={t('deviceSettings.expert.showDeterministicEntropy.description')}
+        displayedValue="Show"
+        extraComponent={<ChevronRightDark />}
+      />
+      { status === 'info' && (
+        <View
+          key="bip85-info"
+          fullscreen
+          verticallyCentered>
+          <ViewHeader title={t('deviceSettings.expert.showDeterministicEntropy.title')} />
+          <ViewContent minHeight="280px">
+            <h1 className="subTitle">What is BIP-85?</h1>
+            <p>
+              This feature is intended to securely derive additional seeds from the BitBox02 wallet.
+
+              This allows the creation of multiple seeds for other applications without the need to store additional backups.
+
+              Multiple seeds can be created by using a different index (positive number) for each application.
+            </p>
+            <h1 className="subTitle">How does this work?</h1>
+            <p>
+              The BitBox02 will ask for which index
+              and for how many words (12, 18 or 24) to create a secret seed.
+
+              The same index and amount of words will create the exact same seed again from this wallet.
+              This is useful in order to re-export the seed in the future,
+              but this also means that the index and amount of words have to be remembered to restore the seed.
+            </p>
+          </ViewContent>
+          <ViewButtons>
+            <Button
+              primary
+              onClick={async () => {
+                setStatus('progress');
+                await invokeBIP85(deviceID);
+                setStatus('idle');
+              }}>
+              Proceed on the BitBox02
+            </Button>
+            <Button
+              secondary
+              onClick={() => setStatus('idle')}>
+              Back
+            </Button>
+          </ViewButtons>
+        </View>
+      )}
+      { status === 'progress' && (
+        <View
+          key="bip85-progress"
+          fullscreen
+          textCenter
+          verticallyCentered>
+          <ViewHeader title={t('deviceSettings.expert.showDeterministicEntropy.title')} />
+          <ViewContent minHeight="280px">
+            <PointToBitBox02 />
+          </ViewContent>
+          <ViewButtons>
+          </ViewButtons>
```

Comment:
what's this

Reply:
> Also relaxed ViewButtons children to be optional, reason: this way
we can keep the component in the DOM even when emtpy so that it
doesn't do a layout shift.

from commit message